### ISSUE TITLE
proxy.bat wrapper does not seem necessary

### ISSUE THREAD

**Issue by lucas-santoni** [clarus-proxy/proxy#7]
When project is built with Maven, a file called `proxy.bat` is placed at the root of the `install/` directory. This file looks like a wrapper to handle options and launch the proxy. However, these options are already handled by the main jar : 

```
$ java -jar ./libs/proxy-main-1.2-SNAPSHOT.jar
usage: java -Djava.ext.dirs=<CLARUS_EXT_DIRS> [PROTOCOL OPTIONS] -jar proxy-1.0-SNAPSHOT.jar [OPTION]... [SERVER_ADDRESS]...
CLARUS extensions:
  <CLARUS_EXT_DIRS>        list the extensions directories that contain protection modules and protocol plugins
Security policy options:
 -sp, --security-policy <PATH>
                           the security policy to apply
Resource consumption options:
 [-mf, --max-frame-len <MAX_FRAME_LENGTH>]
                           maximum frame length to process
 [-lt, --nb-listen-threads <NB_LISTEN_THREADS>]
                           number of listen threads (default: 1). Must be a positive number or the special value 'cores' (number of cores)
 [-st, --nb-session-threads <NB_SESSION_THREADS>]
                           number of session threads (default: number of cores). Must be a positive number or the special value 'cores' (number of cores)
 [-pt, --nb-parser-threads <NB_PARSER_THREADS>]
                           number of parser threads (default: 0). Must be a positive number or 0 or the special value 'cores' (number of cores)
Protocol options:
 [-D<OPTION_NAME>=<OPTION_VALUE>]
                           define options specific to the protocol plugin. Muliple options can be specified
Server addresses:
 <HOSTNAME>[:<PORT>]       server host and optional port. Muliple server addresses can be specified
```

As this file is platform dependant, it could be great to remove it if it is not necessay as it is quite misleading now.

**Comment by lucas-santoni** [comment]
Pushed #8 

Does not actually remove the file as it may have an unkown purpose but recommend to directly run the proxy from command line in the README.

**Comment by lucas-santoni** [comment]
Mostly fixed by #8
